src/lib_c/aarch64-linux-gnu/c/errno.cr

```diff
@@ -1,83 +1,85 @@
-  EINTR           =   4
-  EINVAL          =  22
-  EIO             =   5
-  EISCONN         = 106
-  EISDIR          =  21
-  ELOOP           =  40
-  EMFILE          =  24
-  EMLINK          =  31
-  EMSGSIZE        =  90
-  EMULTIHOP       =  72
-  ENAMETOOLONG    =  36
-  ENETDOWN        = 100
-  ENETRESET       = 102
-  ENETUNREACH     = 101
-  ENFILE          =  23
-  ENOBUFS         = 105
-  ENODEV          =  19
-  ENOENT          =   2
-  ENOEXEC         =   8
-  ENOLCK          =  37
-  ENOLINK         =  67
-  ENOMEM          =  12
-  ENOMSG          =  42
-  ENOPROTOOPT     =  92
-  ENOSPC          =  28
-  ENOSYS          =  38
-  ENOTCONN        = 107
-  ENOTDIR         =  20
-  ENOTEMPTY       =  39
-  ENOTRECOVERABLE = 131
-  ENOTSOCK        =  88
-  ENOTSUP         = LibC::EOPNOTSUPP
-  ENOTTY          =  25
-  ENXIO           =   6
-  EOPNOTSUPP      =  95
-  EOVERFLOW       =  75
-  EOWNERDEAD      = 130
-  EPERM           =   1
-  EPIPE           =  32
-  EPROTO          =  71
-  EPROTONOSUPPORT =  93
-  EPROTOTYPE      =  91
-  ERANGE          =  34
-  EROFS           =  30
-  ESPIPE          =  29
-  ESRCH           =   3
-  ESTALE          = 116
-  ETIMEDOUT       = 110
-  ETXTBSY         =  26
-  EWOULDBLOCK     = LibC::EAGAIN
-  EXDEV           = 18
-  ENODATA         = 61
-  ENOSR           = 63
-  ENOSTR          = 60
-  ETIME           = 62
+  enum Errno
+    E2BIG           =   7
+    EACCES          =  13
+    EADDRINUSE      =  98
+    EADDRNOTAVAIL   =  99
+    EAFNOSUPPORT    =  97
+    EAGAIN          =  11
+    EALREADY        = 114
+    EBADF           =   9
+    EBADMSG         =  74
+    EBUSY           =  16
+    ECANCELED       = 125
+    ECHILD          =  10
+    ECONNABORTED    = 103
+    ECONNREFUSED    = 111
+    ECONNRESET      = 104
+    EDEADLK         =  35
+    EDESTADDRREQ    =  89
+    EDOM            =  33
+    EDQUOT          = 122
+    EEXIST          =  17
+    EFAULT          =  14
+    EFBIG           =  27
+    EHOSTUNREACH    = 113
+    EIDRM           =  43
+    EILSEQ          =  84
+    EINPROGRESS     = 115
+    EINTR           =   4
+    EINVAL          =  22
+    EIO             =   5
+    EISCONN         = 106
+    EISDIR          =  21
+    ELOOP           =  40
+    EMFILE          =  24
+    EMLINK          =  31
+    EMSGSIZE        =  90
+    EMULTIHOP       =  72
+    ENAMETOOLONG    =  36
+    ENETDOWN        = 100
+    ENETRESET       = 102
+    ENETUNREACH     = 101
+    ENFILE          =  23
+    ENOBUFS         = 105
+    ENODEV          =  19
+    ENOENT          =   2
+    ENOEXEC         =   8
+    ENOLCK          =  37
+    ENOLINK         =  67
+    ENOMEM          =  12
+    ENOMSG          =  42
+    ENOPROTOOPT     =  92
+    ENOSPC          =  28
+    ENOSYS          =  38
+    ENOTCONN        = 107
+    ENOTDIR         =  20
+    ENOTEMPTY       =  39
+    ENOTRECOVERABLE = 131
+    ENOTSOCK        =  88
+    ENOTSUP         = LibC::EOPNOTSUPP
```

Comment:
```suggestion
    ENOTSUP         = LibC::Errno::EOPNOTSUPP
```

Ditto on the others

Reply:
Or maybe just `ENOTSUP = EOPNOTSUPP`?

Reply:
It can just be `EOPNOTSUPP`, there's no need to prefix it with `LibC::Errno` because we are already there.

Reply:
ack. Sorry for a lot of noise created due to multiple refreshes of this PR